src/clue.js

```diff
@@ -116,7 +116,52 @@ function pickMystery() {
 
 // ITERATION 3
 
```

Comment:
Muy rebuscado pero loco puntos por  eso 

function revealMystery(mys) {
  return `${mys.suspect.firstName} ${mys.suspect.lastName} killed Mr. Boddy using the ${mys.weapon.name} in the ${mys.room.name}!`;
} donde mys es el objeto que se contruye en picMystery